pt-runner/build.gradle.kts

```diff
@@ -40,13 +40,13 @@ drill {
 
 repositories {
     mavenLocal()
-    maven("https://dl.bintray.com/drill/drill4j")
+    maven("https://drill4j.jfrog.io/artifactory/dril")
 }
 
 
 dependencies {
     compileOnly("org.springframework:spring-context:5.1.8.RELEASE")
-    implementation("org.springframework.samples:spring-petclinic:2.1.0") { isTransitive = false }
+    implementation("org.springframework.samples:spring-petclinic:2.1.0.BUILD-SNAPSHOT") { isTransitive = false }
```

Comment:
any reasons for this change?

Reply:
it's my fault when I published the project to a new repository, I forgot to remove the "build" from the version